lib/spack/spack/test/config.py

```diff
@@ -466,6 +482,8 @@ def test_substitute_tempdir(mock_low_high_config):
     '/path', os.path.sep.join(reps))[:MAX_PADDED_LEN]
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
 @pytest.mark.parametrize('config_settings,expected', [
```

Comment:
I think we'll want to get as many parameterizations of this passing as possible. The `padding`-based ones can be skipped if needed (although they should be split off if they are).

Reply:
They are not passing, will spin them out.